7092513 Samba links liblber from openldap but should not do it.

--- a/source3/configure.in-bkp	2011-09-19 09:34:22.632025100 -0700
+++ b/source3/configure.in	2011-09-19 09:37:16.235176200 -0700
@@ -3285,7 +3285,10 @@
   ##################################################################
   # we might need the lber lib on some systems. To avoid link errors
   # this test must be before the libldap test
-  AC_CHECK_LIB_EXT(lber, LDAP_LIBS, ber_scanf)
+ # The next test should be avoided because libldap60 already have (internal)
+ # ber_scanf but the liblber from openldap is searched first so liblber
+ # is added into LDAP_LIBS
+  #AC_CHECK_LIB_EXT(lber, LDAP_LIBS, ber_scanf)
 
   ########################################################
   # If ber_sockbuf_add_io() is available we can add
